app/src/controllers/plugins/uiExtensions/createImportProps.js

```diff
@@ -231,11 +256,20 @@ export const createImportProps = (pluginName) => ({
     CircleCrossIcon,
     RefreshIcon,
   },
-  utils: { fetch, URLS, debounce, getGroupedDefectTypesOptions, isEmptyObject },
+  utils: {
+    fetch,
+    URLS,
+    debounce,
+    getGroupedDefectTypesOptions,
+    isEmptyObject,
+    getDefaultTestItemLinkParams,
+    getItemNameConfig,
+    getDefectTypeLocators,
+  },
   validators: {
     attributesArray,
     isNotEmptyArray,
     requiredField,
-    helpers: { composeValidators, bindMessageToValidator },
   },
+  helpers: { composeValidators, bindMessageToValidator },
```

Comment:
Why are you moving this `helpers` object up, are you fixing this also from the plugin side to avoid errors? These helpers are part of the validation engine, so it makes sense to store them along with the validation functions.